clang/lib/Sema/SemaType.cpp

```diff
@@ -10067,6 +10069,79 @@ QualType Sema::BuiltinChangeSignedness(QualType BaseType, UTTKind UKind,
   return Context.getQualifiedType(Underlying, BaseType.getQualifiers());
 }
 
+bool Sema::BuiltinIsConvertible(QualType From, QualType To, SourceLocation Loc,
+                                bool CheckNothrow) {
+  if (To->isVoidType())
+    return From->isVoidType();
+
+  if ((!From->isIncompleteArrayType() && !From->isVoidType() &&
```

Comment:
A comment here along the lines of "types must be complete unless they are an incomplete array type or void" also a standard reference would be preferable as well.